pkgs/development/python-modules/virtualenv/default.nix

```diff
@@ -47,10 +54,34 @@ buildPythonPackage rec {
     ./0001-Check-base_prefix-and-base_exec_prefix-for-Python-2.patch
   ];
 
-  meta = {
+  checkInputs = [
+    cython
+    fish
+    flaky
+    pytest-freezegun
+    pytest-mock
+    pytest-timeout
+    pytestCheckHook
+  ];
+
+  preCheck = "export HOME=$(mktemp -d)";
+
+  # Ignore tests which requires network access
+  pytestFlagsArray = [
+    "--ignore tests/unit/create/test_creator.py"
+    "--ignore tests/unit/seed/embed/test_bootstrap_link_via_app_data.py"
+  ];
```

Comment:
Please try out disabledTestFiles. It is the same as disabledTests but for files.

Reply:
It does not work and there is no mention of it in `pkgs/development/interpreters/python/hooks/pytest-check-hook.sh`.

Reply:
It is only available on staging https://github.com/NixOS/nixpkgs/blob/staging/pkgs/development/interpreters/python/hooks/pytest-check-hook.sh#L5

Reply:
@dotlambda you forgot this.

Reply:
It doesn't work. See https://github.com/NixOS/nixpkgs/pull/108529#issuecomment-766361674.